pkg/csi_driver/controller.go

```diff
@@ -101,15 +106,66 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 			return nil, status.Error(codes.AlreadyExists, err.Error())
 		}
 	} else {
+
+		// Get the reserved-IPV4-cidr from the request. If value is not provided, reservedIPV4Range is defaulted to ""
+		reservedIPV4CIDR, ok := req.GetParameters()[paramReservedIPV4CIDR]
+		if ok {
```

Comment:
nit: idiomatic golang style is usually to have a smaller block to handle the error for `if !ok`, then the success case should be not inside a conditional block

Reply:
In this case, you implement some logic in case if the param is present and do nothing in case it is not. So there is just one block depending on the condition.